main.py

```diff
@@ -1,10 +1,60 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main():
-    pass
+    global guild_name
+
+    Player.objects.all().delete()
+    Race.objects.all().delete()
+    Skill.objects.all().delete()
+    Guild.objects.all().delete()
+
+    with open("players.json") as player_data:
+        players = json.load(player_data)
+
+    for player in players:
+        nickname = player
+        email = players[player]["email"]
+        bio = players[player]["bio"]
+        race_data = players[player]["race"]
+        guild_data = players[player]["guild"]
+
+        if not Race.objects.filter(name=race_data["name"]).exists():
+            Race.objects.create(
+                name=race_data["name"],
+                description=race_data["description"]
+            )
+
+        race_name = Race.objects.get(name=race_data["name"])
```

Comment:
Better use just "race" as a variable name (or something else). "race_name" is a bit confusing as you get not a race_name but the race object